salt/cli/support/__init__.py

```diff
@@ -0,0 +1,65 @@
+# coding=utf-8
+'''
+Get default scenario of the support.
+'''
+from __future__ import print_function, unicode_literals, absolute_import
+import yaml
+import os
+import salt.exceptions
+import jinja2
+import logging
+
+log = logging.getLogger(__name__)
+
+
+def _render_profile(path, caller, runner):
+    '''
+    Render profile as Jinja2.
+    :param path:
+    :return:
+    '''
+    env = jinja2.Environment(loader=jinja2.FileSystemLoader(os.path.dirname(path)), trim_blocks=False)
+    return env.get_template(os.path.basename(path)).render(salt=caller, runners=runner).strip()
+
+
+def get_profile(profile, caller, runner):
```

Comment:
This function is a little hard to read. Might be a good target to either break apart or add additional documentation about what's happening here.